content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/log-in-chainguard-enforce/index.md

```diff
@@ -0,0 +1,111 @@
+
+![Chainguard Enforce Console sign in page](console-sign-in.png)
+
+From here, you have three options to authenticate into the Console via one of the following OIDC providers: 
+
+* [Google](https://developers.google.com/identity/openid-connect/openid-connect)
+* GitHub
+* [GitLab](https://docs.gitlab.com/ee/integration/openid_connect_provider.html)
+
+**To authenticate with Google** (including through a Google-based email), click the **Google** button, and then select the specific Google account you would like to use to log into [chainguard.dev](https://chainguard.dev). Note that you may need to log into your Google account. 
+
+![Sign in with Google OIDC](sign-in-with-google.png)
+
+From here, you'll sign into Chainguard Enforce through Google, and will then be brought to the Enforce Console homepage. 
+
+**To authenticate with your GitHub username**, click the **GitHub** button. You may need to log into your GitHub account at this point. When you are logged into GitHub, review the permissions page and confirm that you would like to authorize Chainguard Enforce.
+
+![Authorize Chainguard Enforce on GitHub](sign-in-with-github.png)
+
+If you are satisfied with allowing Chainguard Enforce to have the stated permissions, you'll authorize it by clicking on the **Authorize Chainguard Enforce** button, and then sign in through GitHub. You'll then be brought to the Chainguard Enforce Console homepage.
+
+**To authenticate via GitLab**, you'll be prompted to sign into GitLab if applicable, and then be asked to authorize Chainguard Enforce to use your GitLab account. 
+
+![Sign into GitLab](sign-in-with-gitlab.jpg)
+
+![Authorize Chainguard Enforce to GitLab](authorize-chainguard-enforce-gitlab.png)
+
+As long as you are satisfied with granting Chainguard Enforce the permissions to your GitLab account, you can click on the **Authorize** button. Next, you'll be brought to the Chainguard Enforce Console homepage.
+
+![Chainguard Enfoce Console landing page](chainguard-enforce-console-landing-page.png)
+
+At this point, you are logged into Chainguard Enforce and will have access to your Chainguard Enforce account through the Console. 
+
+## Signing in through `chainctl`
+
+When using `chainctl`, you can initiate the login flow through the [`chainctl auth login`](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainctl-docs/chainctl_auth_login/) command. 
+
+When you run this command, you'll receive output similar to the following on your command prompt.
+
+```
+Authenticating...
+Your browser will now be opened to:
+https://auth.chainguard.dev/authorize?access_type=online&client_id=<RELEVANT ID HERE>
+```
+
+This will trigger a page to open on your default web browser. If this does not occur, you can copy the full URL on your terminal output, and paste it into a browser address bar.
```

Comment:
```suggestion
This will trigger a page to open on your default web browser. If this does not occur, you can copy the full URL on your terminal output and paste it into a browser address bar.
```